packages/core/src/Table/stories/TableRenderers/DropdownColumnRenderer.tsx

```diff
@@ -46,11 +46,6 @@ export const DropdownColumnRenderer = () => {
             return newVal;
           });
           setData(newData);
-        },
-        {
-          popperProps: {
-            disablePortal: true,
-          },
```

Comment:
This seems to be equivalent to what we had previously.
Passing `disablePortal: true` to the Dropdown causes other issues:

![image](https://github.com/lumada-design/hv-uikit-react/assets/638946/15d2190d-5518-4b69-b0b9-2eddb022178a)